client/pages/welcome.jsx

```diff
@@ -16,7 +18,7 @@ export default function Welcome(props) {
         </Row>
         <Row>
           <div className='flex-center'>
-            <button onClick={handleClick} className='green-button margin-3 padding-1'><h6> <i className="bi bi-music-note-beamed white"></i>  Get song recommendations! </h6></button>
+            <button onClick={handleClick} className='green-button margin-3 padding-1'><h6> <i className="bi bi-music-note-beamed white"></i>  Get song recommendations! </h6><a href='#song-form'></a></button>
```

Comment:
Instead of a button with an `onClick` prop, use an anchor tag styled like a button.